raiden-dapp/src/components/UpdateSnackbar.vue

```diff
@@ -13,39 +12,32 @@
 <script lang="ts">
 /* istanbul ignore file */
 import { Component, Vue } from 'vue-property-decorator';
+import { mapGetters } from 'vuex';
 import BlurredOverlay from '@/components/overlays/BlurredOverlay.vue';
 
-@Component({ components: { BlurredOverlay } })
+@Component({
+  components: { BlurredOverlay },
+  computed: {
+    ...mapGetters(['isConnected', 'versionUpdateAvailable']),
+  },
+})
 export default class UpdateSnackbar extends Vue {
+  isConnected!: boolean;
+  versionUpdateAvailable!: boolean;
   isUpdating = false;
-  updateAvailable = false;
-  swRegistration: ServiceWorkerRegistration | null = null;
 
-  created() {
-    window.addEventListener('swUpdated', this.handleSWUpdate, { once: true });
-
-    navigator.serviceWorker.addEventListener('controllerchange', () => {
-      setTimeout(() => {
-        this.updateAvailable = false;
-        window.location.reload();
-      }, 1500);
-    });
-  }
-  beforeDestroy() {
-    window.removeEventListener('swUpdated', this.handleSWUpdate);
-  }
-
-  handleSWUpdate(event: ServiceWorkerUpdatedEvent) {
-    this.swRegistration = event.detail;
-    this.updateAvailable = true;
+  get visible(): boolean {
```

Comment:
What's the reasoning to define `visible`, when you could just use `versionUpdateAvailable`?

Reply:
same here

Reply:
Because at a later point in time I need to extend this for mandatory updates as well.
As I wrote I knew the future of the code. Therefore things sometimes look slightly off in some of the commits. I'm sorry for that. But I barely saw the advantage to optimize each commit as it would be the final one if I already knew what makes sense in terms of the following commits/features.